msg/ko_KR.utf8/utils.msg

```diff
@@ -1221,3 +1221,38 @@ operations:\n\
     -n, --generate-new-key        키 파일 (_keys)에 새로운 키를 추가; 최대 갯수: 128\n\
     -d, --delete-key=KEY_INDEX    키 파일 (_keys)의 한 키를 제거\n\
     -c, --change-key=KEY_INDEX    데이터베이스에 등록된 키를 키 파일 (_keys)에 존재하는 다른 키로 변경\n
+
+$set 58 MSGCAT_UTIL_SET_FLASHBACK
+1 DBA 비밀번호:
+2 잘못된 날짜 형식입니다. 올바른 형식은 dd-mm-yyyy:hh:mm:ss입니다.\n
+3 시작 날짜(%1$s)는 종료 날짜(%2$s)보다 작아야 합니다.\n
+4 현재 날짜 이전(%1$s) 또는 데이터베이스 생성 이후(%2$s) 날짜로 시도하십시오.\n
+5 테이블(%1$s)이 데이터베이스에 없습니다.\n
+6 플래시백할 트랜잭션이 너무 많습니다. 시작 날짜와 종료 날짜 사이의 간격을 줄이십시오.\n
+7 스키마(%1$s)가 변경되었기 때문에 플래시백을 수행할 수 없습니다.\n
+8 로그 볼륨이 제거되었습니다. 나중 날짜로 시도하십시오.\n
+9 출력 파일 '%1$s'을 열 수 없습니다.\n
+10 트랜잭션 ID(%1$d)가 존재하지 않습니다.\n
+11 'conf/cubrid.conf'에 'supplemental_log'를 설정해 주세요. 'supplemental_log'를 설정한 후 실행된 SQL 로그만 플래시백할 수 있습니다.\n
+12 트랜잭션 ID는 %1$d초 안에 입력해야 합니다.\n
```

Comment:
This is optional feedback. I want to know your opinion.

When I read this message compared to the just previous message, I think it would be nice if you add more information like "flashback_time 설정을 통해 변경할 수 있습니다.".